modules/nodes-scheduler-pod-topology-spread-constraints-about.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-scheduler-pod-topology-spread-constraints
+
+[id="nodes-scheduler-pod-topology-spread-constraints-about_{context}"]
+= Understanding pod topology spread constraints
+
+A _pod topology spread constraint_ allows you to provide fine-grained control over the distribution of pods across failure domains to help achieve high availability and more efficient resource utilization.
+
+{product-title} administrators can label nodes to provide topology information, such as regions, zones, nodes, or other user-defined domains. After these labels are set on nodes, users can then define pod topology spread constraints to control the placement of pods across these topology domains.
+
+You specify which pods to group together, which topology domains they should be spread among, and the acceptable skew. Only pods within the same namespace are matched and grouped together when spreading due to a constraint.
+
+// TODO Mention about relationship to affinity/anti-affinity?
```

Comment:
This could be helpful, since prior to PodTopologySpreading the usual advice to get something like this was to use anti-affinity on a pod to itself (the drawback to that is that there could only be one replica on each node, while this allows multiple). In that case it could be worth mentioning Pod Topology Spreading in the affinity/anti-affinity docs too

Reply:
Thanks! I might follow up with adding this in a separate PR, so that we can get this PR merged in time for GA. I'll look at that tomorrow.